addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/data/TankerkoenigService.java

```diff
@@ -43,8 +45,12 @@ private String getResponseString(String apikey, String locationIDs) throws IOExc
         String urlbase = "https://creativecommons.tankerkoenig.de/json/prices.php?";
         String urlcomplete = urlbase + "ids=" + locationIDs + "&apikey=" + apikey;
         try {
+            String userAgent = "OpenHAB, Tankerkoenig-Binding Version ";
```

Comment:
the user agent could start with "openHAB" so it is written the same way as everywhere else.